liquibase-standard/src/main/java/liquibase/command/core/CalculateChecksumCommandStep.java

```diff
@@ -20,22 +31,41 @@ public class CalculateChecksumCommandStep extends AbstractCommandStep {
     protected static final String[] COMMAND_NAME = {"calculateChecksum"};
 
     public static final CommandArgumentDefinition<String> CHANGELOG_FILE_ARG;
-    public static final CommandArgumentDefinition<String> CHANGESET_IDENTIFIER_ARG;
+    public static final CommandArgumentDefinition<String> CHANGESET_PATH_ARG;
+
+    public static final CommandArgumentDefinition<String> CHANGESET_ID_ARG;
+
+    public static final CommandArgumentDefinition<String> CHANGESET_AUTHOR_ARG;
 
     public static final CommandResultDefinition<CheckSum> CHECKSUM_RESULT;
 
-    protected static final int CHANGESET_ID_NUM_PARTS = 3;
-    protected static final int CHANGESET_ID_AUTHOR_PART = 2;
-    protected static final int CHANGESET_ID_CHANGESET_PART = 1;
-    protected static final int CHANGESET_ID_CHANGELOG_PART = 0;
+    public static final CommandArgumentDefinition<String> CHANGESET_IDENTIFIER_ARG;
 
 
     static {
         CommandBuilder builder = new CommandBuilder(COMMAND_NAME);
         CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class).required()
-                .description("The root changelog file").build();
-        CHANGESET_IDENTIFIER_ARG = builder.argument("changesetIdentifier", String.class).required()
-                .description("Changeset ID identifier of form filepath::id::author").build();
+                                    .description("The root changelog file").build();
+
+        CHANGESET_IDENTIFIER_ARG = builder.argument("changeSetIdentifier", String.class)
+                                    .required()
+                                    .description("Changeset ID identifier of form filepath::id::author")
```

Comment:
A minor thing. Can we use `ChangeSet` instead of `Changeset` argument descriptions? In this particular case, I would also remove the ID part as well.